docs/layers/scenegraph-layer.md

```diff
@@ -0,0 +1,159 @@
+
+<p class="badges">
+  <img src="https://img.shields.io/badge/@deck.gl/mesh--layers-lightgrey.svg?style=flat-square" alt="@deck.gl/mesh-layers" />
+  <img src="https://img.shields.io/badge/lighting-TBD-blue.svg?style=flat-square" alt="lighting" />
+</p>
+
+# ScenegraphLayer
+
+The `ScenegraphLayer` renders a number of instances of a complete luma.gl scenegraph. While scenegraphs can be created programmatically, they are typically obtained by loading glTF files.
+
+```js
+import DeckGL from '@deck.gl/react';
+import {ScenegraphLayer} from '@deck.gl/mesh-layers';
+
+const App = ({data, viewport}) => {
+
+  /**
+   * Data format:
+   * [
+   *   {
+   *     position: [-122.45, 37.7],
+   *     angle: 0,
+   *     color: [255, 0, 0]
+   *   },
+   *   {
+   *     position: [-122.46, 37.73],
+   *     angle: 90,
+   *     color: [0, 255, 0]
+   *   },
+   *   ...
+   * ]
+   */
+  const layer = new ScenegraphLayer({
```

Comment:
Example code must be complete, including registering loaders, necessary accessors etc.

Reply:
ok